src/components/notifications.tsx

```
@@ -69,61 +79,135 @@ function NotificationFeed() {
-
-                                    void feed_client.markAsRead(item)
-                                }}
+                                        void feed_client.markAsRead(item)
+                                    }}
+                                >
+                                    {(block.type === 'markdown' || block.type === 'text') && (
+                                        <div
+                                            className={cn(
+                                                item.read_at && 'text-muted-foreground'
+                                            )}
+                                            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)